rules/windows/discovery_remote_system_discovery_commands_windows.toml

```diff
@@ -4,7 +4,11 @@ integration = ["endpoint", "windows"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
+<<<<<<< tune_remote_system_discovery_commands_windows
```

Comment:
I think there are some git comments that stayed when you staged and commited.

Reply:
should be fixed now